packages/hub/readme.md

```diff
@@ -10,9 +10,13 @@ https://devcenter.heroku.com/articles/heroku-cli
 
 ### .env
 
-Copy `.env.example` to `.env`, and make it your own.
+Copy `.env.development` to `.env`, and make it your own.
```

Comment:
I think we would check in both `.env.development` and `.env` to git, right? (`.env.*.local` files should _not_ be checked in)

Reply:
AFAICT we're not using the `.env.*.local` pattern anywhere, at least not yet.  But this should be updated anyway, since we already do check in both `.env` and `.env.development`